lua/entities/gmod_wire_expression2/core/custom/tracesystem.lua

```diff
@@ -1349,7 +1318,7 @@ e2function vector rayCircleIntersection( vector start, vector dir, vector pos, v
 end
 
 e2function vector raySphereIntersection( vector start, vector dir, vector pos, number radius )
-	return RaySphereIntersection( start, dir, pos, radius )
+	return (RaySphereIntersection( start, dir, pos, radius ) or Vector()

```

Comment:
Syntax error?

Reply:
At this point, just make RaySphereIntersection return Vector() instead of nil.

Reply:
Fixed in: https://github.com/wiremod/wire-extras/pull/122/commits/2357f0fa69a3b98433934285d50a878025fe9223